### PR TITLE
Responsive Header

### PR DESCRIPTION
**Pull request by kieranmcnay**
+Created the Header with background image, logo and navigation link placeholders, added hover transitions on the navigation links, Responsiveness based on window size and drop down menu for mobile views

connects to #5

### CODE REVIEW

File: app/styles/navigation.css [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+.navbar-ip, .nav>li>a:focus, .navbar-ip, .nav>li>a:hover {
+    background-color: transparent;
+}
+
+.navbar-ip a:hover, .navbar-ip a:focus {
+    color: rgba(255,255,255,0.8);
+}
+
+a{
```
Comment (TheGarrett) [inline app/styles/navigation.css]: space after the text 'a'.

File: app/styles/navigation.css [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    color: #fff
```
Comment (TheGarrett) [inline app/styles/navigation.css]: could you also not move this into global?

File: app/components/Navigation.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+var React = require('react');
+
+var Navigation = React.createClass({
```
Comment (TheGarrett) [inline app/components/Navigation.js]: Always comment your code so others can easily understand it. This should be done throughout the project

File: app/components/Logo.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+var React = require('react');
+
+var Logo = React.createClass({
+    render: function () {
+        return (
```
Comment (TheGarrett) [inline app/components/Logo.js]: Files should be indented by 2 spaces not 4.

File: app/containers/HeaderContainer.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+                <Logo />
+                 Innovation Program
+                <Navigation />
+            </header>
+
+        );
+    }
+});
+
+
```
Comment (TheGarrett) [inline app/containers/HeaderContainer.js]: Remove 1 of the 2 blank lines

File: app/containers/HeaderContainer.js [hunk 6]
```diff
@@ -0,0 +1,20 @@
+var React = require('react');
+var Logo = require('../components/Logo');
+var Navigation = require('../components/Navigation');
+
```
Comment (TheGarrett) [inline app/containers/HeaderContainer.js]: Remove 1 of the 2 blank lines

File: app/styles/header.css [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+    padding-bottom: 5px;
+    text-shadow: 1px 1px 1px black;
+    font-size: 25px;
+    color: #fff;
+    background: url('/images/banner.png');
+    background-size: cover;
+    min-height: 100px;
+    max-height: 100px;
+    line-height: 50px
+
```
Comment (TheGarrett) [inline app/styles/header.css]: Remove blank line

File: app/components/Hamburger.js [hunk 8]
```diff
@@ -0,0 +1,32 @@
+var React = require('react');
+
+//Code for creating the Navigation buttons on the header container
+var Hamburger = React.createClass({
+
+  // The state of the mobile menu is set to false by default, meaning that it is not showing
+  getInitialState: function(){
```
Comment (Michael-B-Lee) [inline app/components/Hamburger.js]: Space after function parameter

Comment (kieranmcnay) [inline app/components/Hamburger.js]: Done, made change for all functions

**Pull request merged**